mrt_file_server/views.py

```diff
@@ -55,13 +55,13 @@ def upload_single_schematic(file):
   file.filename = secure_filename(file.filename)
   filesize = get_filesize(file)
   
-  if get_file_extension(file.filename) != '.schematic':
+  if get_file_extension(file.filename) != '.schematic' and get_file_extension(file.filename) != '.schem':
```

Comment:
Please extract `get_file_extension()` out into its own variable so that it is not called twice.